backend/blueprints/spa_api/service_layers/player/player.py

```diff
@@ -26,6 +26,10 @@ def create_from_id(id_: str) -> 'Player':
         try:
             steam_profile = get_steam_profile_or_random_response(id_)['response']['players'][0]
         except TypeError:
+            if len(names_and_counts) > 0:
+                return Player(id_=id_, name=names_and_counts[0][0], past_names=names_and_counts, profile_link="",
+                              avatar_link="https://cdn.discordapp.com/attachments/499329447967850507/503993148024815617/781cd87d570a7df1e51994d39dc41b09f1a8cf3a_full.jpg" if not id_.startswith(
```

Comment:
Who owns/uploaded these images?

Reply:
Oops, forgot to replace these with actual images.